functions/common.py

```diff
@@ -27,8 +27,8 @@ def mod(dividend: float, divisor: float) -> float:
     return dividend
 
 
-# Function to convert a whole number to binary
 def decimal_to_binary_integer(value):
+    """Returns a string that contains the binary result of an integer"""
```

Comment:
The docstring convention that we adopted documents the parameters, return values, and possible exceptions thrown in addition to the description of the function itself. I'm not really partial to one convention over another but I would add those things just for the sake of consistency.

Reply:
Thanks I'll change this. Can I change this in the next PR as it's a minor change?

Reply:
Sure as long as we remember it.